.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "myosuite/simhive/myo_sim"]
 	path = myosuite/simhive/myo_sim
-	url = https://github.com/elladyr/myo_sim.git
+	url = https://github.com/MyoHub/myo_sim.git
+	branch = dev
```

Comment:
Note. This needs to be updated before the merge.

Reply:
Updated comments in issues.